docs/src/ActivationCard.doc.js

```diff
@@ -0,0 +1,123 @@
+// @flow strict
+import React, { type Node } from 'react';
+import PropTable from './components/PropTable.js';
+import Example from './components/Example.js';
+import PageHeader from './components/PageHeader.js';
+
+const cards: Array<Node> = [];
+const card = c => cards.push(c);
+
+card(
+  <PageHeader
+    name="ActivationCard"
+    description="Activation cards are used in groups to communicate a user’s stage in a series of steps toward an overall action."
+  />
+);
+
+card(
+  <Example
```

Comment:
<Row gap={2} alignSelf='stretch'>
  <ActivationCard
    status="pending"
    statusMessage="Pending"
    title="Claim your website"
    message="We will notify you via email as soon as your site has been successfully claimed."
    linkData={{href: "https://pinterest.com", label:"Claim your website now"}}
  />
  <ActivationCard
    status="needsAttention"
    statusMessage="Needs attention"
    title="Tag is unhealthy"
    message="Oops! Your tag must be healthy to continue."
    linkData={{accessibilityLabel:"Learn more about how tag health", href: "https://pinterest.com", label:"Learn more"}}
    dismissButton={{
      accessibilityLabel: 'Dismiss card',
      onDismiss: ()=>{},
    }}
  />
</Row>

Reply:
Also, there's an issue with the dismissButton 
![image](https://user-images.githubusercontent.com/10593890/94741706-64865600-0329-11eb-9982-e9ba8ef378e4.png)

Reply:
Check implementation in callout, not happening there 
![image](https://user-images.githubusercontent.com/10593890/94741918-bf1fb200-0329-11eb-8004-99b8860bd707.png)